.github/workflows/build_image.yml

```diff
@@ -0,0 +1,59 @@
+name: Build Container Image
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
```suggestion
```

We don't have a `main` branch and the `release` trigger is more useful in general, I think.